src/jdk.internal.le/share/classes/jdk/internal/org/jline/JdkConsoleProviderImpl.java

```diff
@@ -110,6 +110,8 @@ public char[] readPassword(String fmt, Object ... args) {
                 return jline.readLine(fmt.formatted(args), '\0').toCharArray();
             } catch (EndOfFileException eofe) {
                 return null;
+            } finally {
+                jline.zeroOut();
```

Comment:
The location of the `jline `declaration at new line 134 is unfortunate, but not a problem this PR needs to resolve.  The location makes sense if it was only used in `initJLineIfNeeded()` at some point in the past.

Reply:
Thanks, Iris. Moved those instance fields to the top of the class.